oddt/metrics.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why do we need to declare the encoding? I prefer not to define it, rather remove the UTF8 characters ( I can see hyphens in citations which might easily be replaced by ASCII chars).

Reply:
I don't know why we need encoding, you've written this line :) I think it was added automatically at some point, lets just remove it and fix all non-ascii characters